docs/docsite/rst/scenario_guides/guide_docker.rst

```diff
@@ -155,176 +104,124 @@ running Ansible:
 
-        - name: Test docker_inventory, this will not connect to any hosts
-          hosts: all
-          gather_facts: no
-          tasks:
-            - debug:
-                msg: "Container - {{ inventory_hostname }}"
+    docker_image_info module
+        The :ref:`community.docker.docker_image_info module <ansible_collections.community.docker.docker_image_info_module>` allows to list and inspect images.
 
-Configuration
-.............
-You can control the behavior of the inventory script by defining environment variables, or
-creating a docker.yml file (sample provided in https://raw.githubusercontent.com/ansible-collections/community.general/main/scripts/inventory/docker.py). The order of precedence is the docker.yml
-file and then environment variables.
+    docker_network module
+        The :ref:`community.docker.docker_network module <ansible_collections.community.docker.docker_network_module>` provides full control over Docker networks.
 
+    docker_network_info module
+        The :ref:`community.docker.docker_network_info module <ansible_collections.community.docker.docker_network_info_module>` allows to inspect Docker networks.
 
-Environment Variables
-;;;;;;;;;;;;;;;;;;;;;;
+    docker_volume_info module
+        The :ref:`community.docker.docker_volume_info module <ansible_collections.community.docker.docker_volume_info_module>` provides full control over Docker volumes.
 
-To connect to a single Docker API the following variables can be defined in the environment to control the connection
-options. These are the same environment variables used by the Docker modules.
+    docker_volume module
+        The :ref:`community.docker.docker_volume module <ansible_collections.community.docker.docker_volume_module>` allows to inspect Docker volumes.
 
-    DOCKER_HOST
-        The URL or Unix socket path used to connect to the Docker API. Defaults to unix://var/run/docker.sock.
+    docker_container module
+        The :ref:`community.docker.docker_container module <ansible_collections.community.docker.docker_container_module>` manages the container lifecycle by providing the ability to create, update, stop, start and destroy a Docker container.
 
-    DOCKER_API_VERSION:
-        The version of the Docker API running on the Docker Host. Defaults to the latest version of the API supported
-        by docker-py.
+    docker_container_info module
+        The :ref:`community.docker.docker_container_info module <ansible_collections.community.docker.docker_container_info_module>` allows to inspect a Docker container.
 
-    DOCKER_TIMEOUT:
-        The maximum amount of time in seconds to wait on a response from the API. Defaults to 60 seconds.
 
-    DOCKER_TLS:
-        Secure the connection to the API by using TLS without verifying the authenticity of the Docker host server.
-        Defaults to False.
+Docker Compose
```

Comment:
```suggestion
Docker compose
```

Reply:
Docker Compose (with this capitalization) is the product name (https://en.wikipedia.org/wiki/Docker_(software)#Tools).

Reply:
good catch!